### PR TITLE
Change integration test to use operation progress

### PR DESCRIPTION
**Pull request by davkean**
This changes the integration test to use operation progress instead of the NuGet restore service. This is a better indication of when the project system is done.

### CODE REVIEW

File: build/Packages.targets [hunk 1]
```diff
@@ -122,7 +122,7 @@
     <PackageReference Update="Microsoft.Build.Tasks.Core" Version="16.0.461"/>
 
     <!-- Libraries -->
-    <PackageReference Update="Newtonsoft.Json" Version="9.0.1"/>
+    <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)"/>
```
Comment (davkean) [inline build/Packages.targets]: This is needed as Apex has dependency on a higher version than VS ships.

**Comment by drewnoakes** [comment]
I'm pretty sure the integration tests branch doesn't need those deleted files.

File: src/HostAgnostic.props [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <Project>
   <PropertyGroup>
     <BannedSymbolsOptOut Condition="$(IsTestProject)">true</BannedSymbolsOptOut>
+    <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
```
Comment (davidwengier) [inline src/HostAgnostic.props]: As of 16.2, VS ships with 12.0.2 and we're supposed to update "at your convenience", so can we just set this to 12.0.2 and not need the special case for integration tests?

**Comment by davkean** [comment]
To avoid getting out of sync, can we merge that branch back into mainsteam and just not turn them on?

File: src/HostAgnostic.props (hunk 2 above)
Comment (davkean) [reply src/HostAgnostic.props]: Ah, good to know, I'll do a followup change. I need to make sure we don't have a handshake with other components here.

**Comment by drewnoakes** [comment]
> can we merge that branch back into mainsteam and just not turn them on?

Fine with me. @jmarolf what do you think?

File: src/HostAgnostic.props (hunk 2 above)
Comment (davkean) [reply src/HostAgnostic.props]: https://github.com/dotnet/project-system/pull/5079

**Pull request merged**